src/util.ts

```diff
@@ -436,15 +436,15 @@ function generateTemplateFile(application: string, type: string, id: string, ver
 
       const defaultTemplateName = applicationProperties[application][type].templateName;
       const webExtensionPath = applicationProperties[application][type].webExtensionPath;
-      let templatePath = path.join(process.cwd(), defaultTemplateName);
-
-      let i = 0;
-      while (fs.existsSync(templatePath)) {
-        const [templateName, templateExtension] = defaultTemplateName.split('.');
-        templatePath = path.join(process.cwd(), templateName + i + '.' + templateExtension);
-        i++;
+      const extension = path.extname(defaultTemplateName);
+      const templatePath = path.join(os.tmpdir(), `${application} add-in ${id}${extension}`);
+  
+      if (fs.existsSync(templatePath)) { 
+        fs.removeSync(templatePath);
       }
 
+      fs.ensureDirSync(path.dirname(templatePath));
```

Comment:
I added this check in case the templatePath uses a subdirectory in the path. This could be added so that the sideload document files are in one directory such as "%TEMP%\office-toolbox-sideload". I'd like input if that would be a good addition, because then you could more easily delete that directory if desired. We don't have a way to cleanup these temp files. I was thinking that "npm stop" could invoke an office-toolbox command which would clean up, e.g. office-toolbox sideload-cleanup.